app/src/main/res/layout/activity_my.xml

```diff
@@ -0,0 +1,62 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
By looking at the name of this xml file, I have no idea what activity this is for. Please have the names be reflective of what purpose it serves. If this is the add stash xml, just call it 'addStash.xml'